modules/react/src/components/export-video/export-video-panel-settings.js

```diff
@@ -77,11 +68,18 @@ const InputGrid = styled.div`
   grid-row-gap: ${DEFAULT_ROW_GAP};
 `;
 
+const getOptionValue = r => r.value;
+const displayOption = r => r.label;
+const getSelectedItems = (options, value) => {
+  const item = options.find(o => o.value === value);
+  return item ? [item] : [];
```

Comment:
Can you help me understand the benefit of using a list here? Is it a pattern to gracefully handle an unknown selection?

Reply:
The api to ItemSelector has no official docs afaict, so I went with this as the prop accepts an array.

On closer inspection of item-selector.js, it could be sufficient to just return `options.find(..)` directly and have the component upcast the undefined to an empty array. It seems to call `utils::toArray`.